inc/ticket.class.php

```diff
@@ -4891,6 +5050,26 @@ function showForm($ID, $options = []) {
       }
       echo "</tr>";
 
+      if ($this->isNewItem() && !$tt->isHiddenField('_contracts_id')) {
+         echo "<tr class='tab_bg_1'>";
+         echo "<th>".$tt->getBeginHiddenFieldText('_contracts_id');
+         printf(__('%1$s%2$s'), Contract::getTypeName(1), $tt->getMandatoryMark('_contracts_id'));
+         echo $tt->getEndHiddenFieldText('_contracts_id')."</th>";
+         echo "<td>";
+         echo $tt->getBeginHiddenFieldValue('_contracts_id');
+
+         $contract_value = Entity::getDefaultContract($options['entities_id'] ?? $this->fields['entities_id']) ?: $tt->predefined['_contracts_id'] ?? 0;
```

Comment:
IMHO, if the ticket template provides a predefined value for `_contracts_id`, it should be used prior to the default value defined in entity.

Reply:
Done in a5da574 .